sdk/networkcloud/Azure.ResourceManager.NetworkCloud/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.1.0 (2025-01-02)
+
+- This is a stable SDK version based on the 2024-07-01 API.
+
 ## 1.1.0-beta.2 (Unreleased)
```

Comment:
please remove the beta .2 block

Reply:
I had removed that block, but then got an error in CI on the changelog checks. I replaced it to resolve that issue. is there another place that should be updated with the removal?

Reply:
> I had removed that block, but then got an error in CI on the changelog checks. I replaced it to resolve that issue. is there another place that should be updated with the removal?

try moving it above the 1.1.0 and call it 1.2.0-beat.1 (Unreleased)